### PR TITLE
style(website): visual tweaks

### PR DESCRIPTION
**Pull request by peduarte**
Changes by @colmtuite

### CODE REVIEW

File: packages/website/src/templates/doc-page.js [hunk 1]
```diff
@@ -35,9 +48,18 @@ function DocPageTemplate({ data, location, ...props }) {
...
             {data.mdx.frontmatter.title}
           </RC.Heading>
+          <RC.Heading
+            size={1}
+            mb={7}
+            as="h2"
+            lineHeight={2}
+            style={{ color: 'hsl(209, 14%, 46%)' }}
+          >
+            This is the description pulled from the primitive. It can be edited here by the user.
```
Comment (peduarte) [inline packages/website/src/templates/doc-page.js]: This is currently showing on all pages. A bit weird 🤔 

Want me to move it into the `mdx` files? Could be a metadata on them, or could just be a `##` (`h2`)

File: packages/website/src/docs/getting-started.mdx [hunk 2]
```diff
@@ -3,57 +3,36 @@ title: Getting Started
...
 
-Radix is currently under heavy development, but if you'd like to use it anyway:
+Radix is currently under heavy development, but can install it by running either of these commands:
 
 ```js
-# npm
 npm install @modulz/radix
 ```
 
+or
```
Comment (peduarte) [inline packages/website/src/docs/getting-started.mdx]: image

Need a bit of spacing above the `or`

File: packages/website/src/docs/getting-started.mdx [hunk 3]
```diff
@@ -3,57 +3,36 @@ title: Getting Started
...
 ```js
-# yarn
 yarn add @modulz/radix
 ```
 
-### Wrap it in `RadixProvider`
+### RadixProvider
 
-Make sure to wrap your app in `RadixProvider` to get the default theme
+Make sure to wrap your app in `RadixProvider to get the default theme
```
Comment (peduarte) [inline packages/website/src/docs/getting-started.mdx]: missing closing backtick after `RadixProvider`

File: packages/website/src/index.js [hunk 4]
```diff
@@ -53,25 +53,48 @@ export const wrapPageElement = ({ element, props }) => {
               width={['100%', 200, 250]}
               height={['auto', '100vh']}
               overflow={['auto', 'scroll']}
-              pt={4}
-              pb={[0, 4]}
+              pb={[0, 8]}
```
Comment (peduarte) [inline packages/website/src/index.js]: 👍

File: packages/website/src/index.js [hunk 5]
```diff
@@ -53,25 +53,48 @@ export const wrapPageElement = ({ element, props }) => {
...
-              style={{ WebkitOverflowScrolling: 'touch' }}
+              style={{ WebkitOverflowScrolling: 'touch', overflowX: 'hidden' }}
             >
-              <Heading size={1} ml={5} mb={4} fontWeight="bold">
+              <Box py={2} px={4}>
                 <Flex alignItems="center">
-                  Radix{' '}
-                  <Badge ml={2} variant="gray">
+                  <Box>
+                    <svg
```
Comment (peduarte) [inline packages/website/src/index.js]: Hmm, finding it a bit odd just showing the Modulz logo, then Alpha next to it.

Why did you remove Radix?

File: packages/website/src/templates/doc-page.js [hunk 6]
```diff
@@ -24,7 +24,20 @@ export const components = {
...
+  ),
+  p: props => (
+    <RC.Text
+      {...props}
+      as="p"
+      size={3}
+      mt={0}
+      mb={0}
+      lineHeight={1}
+      style={{ color: 'hsl(208, 14%, 46%)' }}
```
Comment (peduarte) [inline packages/website/src/templates/doc-page.js]: What's this inline color? The Radix `Text` Component accepts a `textColor` prop which is bound to the theme.

File: packages/website/src/templates/doc-page.js [hunk 7]
```diff
@@ -35,9 +48,18 @@ function DocPageTemplate({ data, location, ...props }) {
...
-          <RC.Heading size={3} fontWeight="bold">
+          <RC.Heading size={4} fontWeight="bold" mb={2} lineHeight={3}>
             {data.mdx.frontmatter.title}
           </RC.Heading>
+          <RC.Heading
+            size={1}
+            mb={7}
+            as="h2"
+            lineHeight={2}
+            style={{ color: 'hsl(209, 14%, 46%)' }}
```
Comment (peduarte) [inline packages/website/src/templates/doc-page.js]: What's this inline color? The Radix Text Component accepts a `textColor` prop which is bound to the theme.

File: packages/website/src/docs/getting-started.mdx [hunk 8]
```diff
@@ -3,57 +3,36 @@ title: Getting Started
...
-
 Radix is a Component Library built on top of [Radix System](/docs/radix-system).
 
-<Divider size={2} my={6} />
-
-### Install it
+### Installing Radix
 
-Radix is currently under heavy development, but if you'd like to use it anyway:
+Radix is currently under heavy development, but can install it by running either of these commands:
```
Comment (peduarte) [inline packages/website/src/docs/getting-started.mdx]: Should be "can be installed by running ..."

File: packages/website/src/templates/doc-page.js (hunk 1 above)
Comment (colmtuite) [reply packages/website/src/templates/doc-page.js]: Sure. Eventually, it should be pulled from the primitive file by default, then editable in the GUI.

File: packages/website/src/docs/getting-started.mdx (hunk 2 above)
Comment (colmtuite) [reply packages/website/src/docs/getting-started.mdx]: That's fine for now. I want to optimise for the making the component pages gorgeous first. Then I'll loop back around and fix things in a more systematic way.

File: packages/website/src/index.js (hunk 5 above)
Comment (colmtuite) [reply packages/website/src/index.js]: Our customers will use their company logo. We can't expect every customer to design a separate logo for their design system. Our styleguide should be 100% Modulz generated, no special treatment.

We don't have time or resources right now to design a Radix brand, even our Modulz brand is rough.

We could remove the alpha badge though and put it elsewhere.

File: packages/website/src/templates/doc-page.js (hunk 6 above)
Comment (colmtuite) [reply packages/website/src/templates/doc-page.js]: I tried using `color={grays.5}` but it didn't work obviously. I'll fix it.

File: packages/website/src/templates/doc-page.js (hunk 7 above)
Comment (colmtuite) [reply packages/website/src/templates/doc-page.js]: I tried using color={grays.5} but it didn't work obviously. I'll fix it.

File: packages/website/src/components/CodeBlock.js [hunk 9]
```diff
@@ -89,20 +89,20 @@ export default ({ children, className, live, render, ghost }) => {
...
+              border: `1px solid ${colors.grays[3]}`,
+              borderRadius: '5px 5px 0 0',
             }}
           />
           <LiveEditor
             padding={radixTheme.space[4]}
             style={{
-              borderRadius: '0 0 3px 3px',
-              border: `1px solid ${colors.grays[2]}`,
+              borderRadius: '0 0 5px 5px',
```
Comment (peduarte) [inline packages/website/src/components/CodeBlock.js]: We should be able to read `theme.radii` here

**Comment by peduarte** [comment]
Overall, these design tweaks are soooo much better 🔥
